packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -141,35 +149,70 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 	return (
 		<Stack className="container-summary-view">
 			<StackItem>
-				<span>
-					<b>Container</b>:{" "}
-					{containerState.nickname !== undefined
-						? `${containerState.nickname} (${containerState.id})`
-						: containerState.id}
-				</span>
-			</StackItem>
-			<StackItem>
-				<span>
-					<b>Status</b>: {statusString}
-				</span>
-			</StackItem>
-			<StackItem>
-				{containerState.clientId === undefined ? (
-					<></>
-				) : (
-					<span>
-						<b>Client ID</b>: {containerState.clientId}
-					</span>
-				)}
-			</StackItem>
-			<StackItem>
-				{containerState.audienceId === undefined ? (
-					<></>
-				) : (
-					<span>
-						<b>Audience ID</b>: {containerState.audienceId}
-					</span>
-				)}
+				<Table>
+					<TableBody>
+						<TableRow>
+							<TableCell>
+								<TableCellLayout>
+									<b>Container</b>:
+								</TableCellLayout>
+							</TableCell>
+							<TableCell>
+								<TableCellLayout>
+									{containerState.nickname !== undefined
+										? `${containerState.nickname} (${containerState.id})`
+										: containerState.id}
+								</TableCellLayout>
+							</TableCell>
+						</TableRow>
+						<TableRow>
+							<TableCell>
+								<TableCellLayout>
+									<b>Status</b>:
+								</TableCellLayout>
+							</TableCell>
+							<TableCell>
+								<TableCellLayout>
+									{((): JSX.Element => {
+										switch (statusString) {
+											case "disconnected":
+												return <Badge color="danger">{statusString}</Badge>;
+											case "detached":
+												return (
+													<Badge color="warning">{statusString}</Badge>
+												);
+											default:
+												return (
+													<Badge color="success">{statusString}</Badge>
```

Comment:
I think we might be better off splitting the status into 2 labels. Open to other thoughts/inputs but it would be better to split the status string (or not combine them to begin with) and individually color code each status.

Reply:
I agree.